lib/nylas/room_resource.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Nylas
+  # Ruby representation of a Nylas Room Resource object
+  # @see https://developer.nylas.com/docs/api/#tag--Room-Resources
+  class RoomResource
+    include Model
+    self.resources_path = "/resources"
+    allows_operations(listable: true, showable: true)
```

Comment:
Can it be `showable`? I don't see any route to fetch a particular room. I might be looking at wrong place.